### ISSUE TITLE
Accessibility Concerns

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#24]
Hello! I hope this message finds you well. I wanted to bring to your attention that the current version of the study group map seems to be missing some accessibility features. It would be great if we could make it more accessible to everyone. Thanks for your understanding and keep up the great work!
